protocol/message.go

```diff
@@ -29,6 +30,11 @@ type MessageBase struct {
 	MessageType uint8
 }
 
+// ClearCbor removes the original CBOR that was parsed
+func (m *MessageBase) ClearCbor() {
+	m.rawCbor = nil
+}
```

Comment:
Instead of adding a new function for this, it would be cleaner to have `SetCbor()` properly handle `nil` instead of creating an empty slice.